iceoryx_dust/cli/include/iox/cli/arguments.inl

```diff
@@ -16,14 +16,12 @@
 #ifndef IOX_DUST_CLI_COMMAND_LINE_OPTION_VALUE_INL
 #define IOX_DUST_CLI_COMMAND_LINE_OPTION_VALUE_INL
```

Comment:
This include guard/macro name is outdated. Even though not due to your changes, can you please update it if there is something else to commit?